vllm/model_executor/models/commandr.py

```diff
@@ -206,7 +215,8 @@ def forward(
         q, k, v = qkv.split([self.q_size, self.kv_size, self.kv_size], dim=-1)
         if self.use_qk_norm:
             q, k = self._apply_qk_norm(q, k)
-        q, k = self.rotary_emb(positions, q, k)
+        if self.sliding_window:
+            q, k = self.rotary_emb(positions, q, k)
```

Comment:
I think you inverted the condition here? Since Cohere (v1) doesn't have sliding window.

Reply:
You're right, I'll fix this. The condition is ok for v2 but should not be there at all for v1.

Reply:
@DarkLight1337 I have added the extra check.